packages/babel-plugin-proposal-decorators/test/fixtures/2021-12-accessors--to-es2015/context-name/output.js

```diff
@@ -76,46 +73,37 @@ function _get_a2() {
 }
 (() => {
   [_init_a, _init_a2, _get_a, _set_a, _init_computedKey, _init_computedKey2, _init_computedKey3, _init_computedKey4, _init_computedKey5, _init_computedKey6, _init_computedKey7, _initStatic] = babelHelpers.applyDecs(_Foo, [[dec, 6, "a"], [dec, 6, "a", function () {
-    return babelHelpers.classStaticPrivateFieldSpecGet(this, _Foo, _B);
+    return babelHelpers.classCheckPrivateStaticAccess(this, _Foo, _B)._;
   }, function (value) {
-    babelHelpers.classStaticPrivateFieldSpecSet(this, _Foo, _B, value);
+    _B._ = babelHelpers.classCheckPrivateStaticAccess(this, _Foo, value);
   }], [dec, 6, "b"], [dec, 6, "c"], [dec, 6, 0], [dec, 6, 1], [dec, 6, 2n], [dec, 6, 3n], [dec, 6, _computedKey]], []);
   _initStatic(_Foo);
 })();
 var _A = {
-  writable: true,
-  value: _init_a(_Foo)
+  _: _init_a(_Foo)
 };
```

Comment:
We still need the object in this case because we need to differentiate between "static private field initialized to undefined" and "static private field not initialized". Using an object like this means that we get the TypeError "for free" in the latter case, because `_A._` throws.

This is observable in the following example:
```js
class A {
  static #x = A.#y;
  static #y = 1;
}
```

Reply:
Now that we have used an additional object, can we use something similar to `usingCtx`.
Then we can initialize this object and `WeakSet` in the helper, and at the same time merge the `set/get` helper. I believe anyone needs to use `get/set` at the same time.

Reply:
The object and the weakset are never used together: the object is for static private fields, while the weakset is for instance private methods and accessors.